code/modules/research/xenobiology/crossbreeding/stabilized.dm

```diff
@@ -142,18 +161,23 @@ Stabilized extracts:
 
 /obj/item/slimecross/stabilized/oil
 	colour = "oil"
+	effect_desc = "The owner will violently explode when they die while holding this extract."
 
 /obj/item/slimecross/stabilized/black
 	colour = "black"
+	effect_desc = "While strangling someone, the owner's hands melt around their neck, draining their life in exchange for food and healing."
 
 /obj/item/slimecross/stabilized/lightpink
 	colour = "light pink"
+	effect_desc = "The owner moves at high speeds while holding this extract, also stabilizes anyone in critical condition around you using Epinephrine."
 
 /obj/item/slimecross/stabilized/adamantine
 	colour = "adamantine"
+	effect_desc = "Owner gains a slight boost in damage resistance to all types."
 
 /obj/item/slimecross/stabilized/rainbow
 	colour = "rainbow"
+	effect_desc = "Accepts a regenerative extract, automatically uses it if the owner goes into a critical condition."
```

Comment:
Accepts a regenerative extract and automatically uses it if the owner enters a critical condition.